packages/cmr-client/src/CMR.ts

```diff
@@ -219,20 +256,24 @@ class CMR {
     const granuleId = ummgMetadata.GranuleUR || 'no GranuleId found on input metadata';
     logDetails.granuleId = granuleId;
 
-    let response;
+    let response: {
+      body: {
+        errors?: unknown
+      }
+    };
     try {
       await validateUMMG(ummgMetadata, granuleId, this.provider);
 
       response = await got.put(
-        `${getUrl('ingest', this.provider)}granules/${granuleId}`,
+        `${getIngestUrl({ provider: this.provider })}granules/${granuleId}`,
         {
-          json: true,
-          body: ummgMetadata,
+          json: ummgMetadata,
```

Comment:
These request properties certainly seem different, so I assume that is intentional?

Reply:
Yes, that is a result of updating the version of `got`.